Remove redundant, language-specific aliases for KC_ALGR

vomindoraan opened the pull request:
This PR continues from where #4389 left off. Namely, it removes redundant, language-specific aliases for `KC_ALGR`, since that keycode is standard now (see the aforementioned PR). This is done to improve consistency throughout keymaps and the codebase as a whole.

## Description
There is no national layout<sup>1</sup> in which AltGr is anything other than right Alt. Therefore, language-specific aliases for `KC_ALGR` are unnecessary. This PR removes `DE_ALGR`, `FR_ALGR`, `FR_CH_ALGR` etc. and replaces uses thereof with `KC_ALGR`. Also, two of the language keymaps defined the same <code>AL**T**GR(kc)</code> alias for `ALGR(kc)`, which I've replaced as well.

<sub><sup>1</sup> Other than maybe the German NEO layout, but that one has its own modifiers (`NEO_L1_L`, `NEO_L1_R`, `NEO_L2_L`, `NEO_L2_R`), which I haven't touched.</sub>

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [x] Enhancement/Optimization
- [x] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

drashna commented:
Honestly, I think we should keep the language specific keycodes, but they should be using the correct KC_ versions. 

That way, they fit better into the keymap_extras stuff.

vomindoraan commented:
It's usually not done for any of the other modifiers, though, nor is it done for keycodes like `KC_SPACE` etc. The `KC_ALGR` aliases are a holdover from before `KC_ALGR` was added as a standard keycode, and every extra keymap defined it separately. Therefore, I think it would be okay to remove them.

Speaking of which, I just noticed [these](https://github.com/qmk/qmk_firmware/pull/4720/files#diff-a5cfba5fcc54eb14cc438356f19459a4L21); they should probably be removed too for the same reason.

vomindoraan commented:
Updated to latest master, removed two more redundant aliases (`BE_LALT` and `BE_LGUI`).

The plethora of redundant [aliases](https://github.com/qmk/qmk_firmware/blob/40383089d035f69101851831f756508271fff103/quantum/keymap_extras/keymap_uk.h#L73-L178) in the UK keymap can stay for now.
